package.json

```diff
@@ -29,9 +29,11 @@
     "vue": "^2.6.14"
   },
   "devDependencies": {
-    "@storybook/addons": "^6.3.9",
-    "@storybook/client-api": "^6.3.9",
+    "@storybook/addons": "^6.4.14",
+    "@storybook/client-api": "^6.4.14",
+    "@storybook/csf": "^0.0.2--canary.87bc651.0",
```

Comment:
This is not desirable I admit. However, even the Storybook uses this version for their latest versions. `0.0.1` is published 2 years ago and doesn't have `AnyFramework` interface.